app/controllers/concerns/manifest_params.rb

```diff
@@ -1,5 +1,12 @@
 module ManifestParams
+  def deep_reject!(hash, &block)
```

Comment:
good idea. I will write a unit test for it.

The existing feature tests will fail without it though; the web form submits lots of empty fields.